tests/test_omxplayer.py

```diff
@@ -204,7 +211,7 @@ def patch_interface_and_run_command(self, interface_name,
             self.run_command(command_name, *command_args)
             # generates a call of the form `call().CanQuit`
             expected_call = getattr(call(), interface_command_name)(*expected_args)
-            interface.assert_has_calls(expected_call)
```

Comment:
I don't know why this ever worked, but it did, the parameter `calls` should be a sequence.